MaxText/configs/base.yml

```diff
@@ -371,4 +371,6 @@ dataset_directory: "/mnt/gcsfuse"
 gcs_metrics_bucket: "distributed-training-metrics"
 ### Parameters used for GCSFuse
 # The GCS bucket that holds the dataset.
-dataset_bucket: xai-hf-dataset-parquet
\ No newline at end of file
+dataset_bucket: xai-hf-dataset-parquet
+# The strategy used for reading the dataset.
```

Comment:
nit: point to the location that lists the enums of these strategies.

Reply:
Done.